nflow-engine/src/test/java/io/nflow/engine/service/WorkflowDefinitionServiceWithSpringTest.java

```diff
@@ -50,20 +50,22 @@
 @DirtiesContext
 public class WorkflowDefinitionServiceWithSpringTest {
 
+  static WorkflowDefinitionService workflowDefinitionService;
```

Comment:
You could try injecting (or @Autowired) the bean here, like WorkflowDefinitionSpringBeanScanner is done at line 160?

```
@Inject
private WorkflowDefinitionService workflowDefinitionService;
```

Reply:
Done